client/sites/components/site-preview-pane/dotcom-preview-pane.tsx

```diff
@@ -81,6 +82,18 @@ const DotcomPreviewPane = ( {
 	const isPlanExpired = !! site.plan?.expired;
 	const isMigrationPending = getMigrationStatus( site ) === 'pending';
 
+	const [ isDuplicateViewsExperiment, setIsDuplicateViewsExperiment ] = useState( false );
+	useEffect( () => {
+		const experimentName = 'calypso_post_onboarding_holdout_160125';
+		( async () => {
+			const duplicateViewsExperimentAssignment = await loadExperimentAssignment( experimentName );
+			if ( duplicateViewsExperimentAssignment.variationName === 'treatment' ) {
+				console.log( 'You are in the duplicate views experiment' );
+				setIsDuplicateViewsExperiment( true );
+			}
+		} )();
+	}, [] );
```

Comment:
We should refactor this into a common function and replace `const isDuplicateViewsExperiment = true;`